medical_patient_disease/views/medical_patient_disease_view.xml

```diff
@@ -162,7 +162,7 @@
                             </group>
                         </group>
                     </group>
-                    <group name="state" attrs="{'invisible': [('id','=',False)]}">
```

Comment:
I have been fighting a lot with this `trying to customize view display in creation mode` but I can't remember that I ever found a solution.
I'll let you know if I can remember/find something.